pkg/beacon/analyzer.go

```diff
@@ -288,3 +303,121 @@ func countAndRemoveConsecutiveDuplicates(numberList []int64) ([]int64, map[int64
 	}
 	return result, counts
 }
+
+//getTsHistogramScore
+func getTsHistogramScore(min int64, max int64, tsList []int64) ([]int64, []int, map[int]int, float64) {
+
+	// get bucket list
+	// we currently look at a 24 hour period
+	bucketList := createBuckets(min, max, 24)
```

Comment:
I think we may want to pass 25 here or add 1 in the function. It looks like the array returned denotes the endpoints of the buckets. If we want 24 buckets (or in other words, the length of `freqList` to be 24), we need to have 25 endpoints.